<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fruitandveg_filter</class>
 <widget class="QDialog" name="fruitandveg_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(132, 169, 172);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
   </property>
   <property name="title">
    <string>آجیل ، میوه و تره بار</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
        </property>
        <property name="text">
         <string>نام محصول:  </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
        </property>
        <property name="text">
         <string>تومان</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
        </property>
        <property name="text">
         <string>تا</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
        </property>
        <property name="text">
         <string>قیمت(کیلویی) :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
        </property>
        <property name="text">
         <string>نام فروشنده:  </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);</string>
        </property>
        <property name="text">
         <string>کلمات کلیدی:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
      </property>
      <property name="text">
       <string>حذف فیلتر</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 64, 81);
font: 12pt &quot;Lalezar&quot;;</string>
      </property>
      <property name="text">
       <string>فیلتر کردن</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
